Ginger/GingerCoreNET/RunLib/CLILib/CLIDynamicFile.cs

```diff
@@ -217,7 +217,7 @@ private void LoadALMDetailsFromJSON(GingerExecConfig gingerExecConfig)
             foreach (AlmDetails almDetails in gingerExecConfig.AlmsDetails)
             {
                 ALMIntegrationEnums.eALMType almTypeToConfigure;
-                if (Enum.TryParse<ALMIntegrationEnums.eALMType>(almDetails.ALMType, out almTypeToConfigure))
+                if (Enum.TryParse(almDetails.ALMType.ToString(), out almTypeToConfigure))
```

Comment:
why this change was needed

Reply:
Because almDetails.ALMType property's type is changed from string to enum